src/devices/machine/smc91c9x.cpp

```diff
@@ -181,25 +214,210 @@ void smc91c9x_device::device_reset()
 	m_reg[EREG_ERCV]         = 0x331f;   m_regmask[EREG_ERCV]         = 0x009f;
 
 	update_ethernet_irq();
-	m_tx_timer->adjust(attotime::never);
 }
 
-
```

Comment:
The double blank line here was to give a bigger visual break between classes than between methods.  Why remove it?

Reply:
Done in the new PR.